java/src/jmri/jmrix/configurexml/AbstractConfigXmlTestBase.java

```diff
@@ -0,0 +1,5 @@
+package jmri.jmrix.configurexml;
+
+public class AbstractConfigXmlTestBase {
```

Comment:
This class is in the src tree but I think it should be in the test tree.
 java/src/jmri/jmrix/configurexml/AbstractConfigXmlTestBase.java
should be
 java/test/jmri/jmrix/configurexml/AbstractConfigXmlTestBase.java

Reply:
Thanks for spotting it - it wasn't supposed to be there at all. An autocreated file because I hit the wrong option when I was writing a test :p